src/app/cases/page.tsx

```diff
@@ -85,162 +57,78 @@ export default function Page() {
       caseData
         .filter(
           c =>
-            caseFilters.remote.size === 0 ||
-            (caseFilters.remote.has('Remote') && c.is_remote) ||
-            (caseFilters.remote.has('In Person') && !c.is_remote),
+            remoteFilters.size === 0 ||
+            (remoteFilters.has('Remote') && c.is_remote) ||
+            (remoteFilters.has('In Person') && !c.is_remote),
         )
         .filter(
           c =>
-            caseFilters.role.size === 0 ||
-            (caseFilters.role.has('Interpreter') && c.needs_interpreter) ||
-            (caseFilters.role.has('Attorney') && c.needs_attorney),
+            languagesFilters.size === 0 ||
+            c.languages.find(l => languagesFilters.has(l)),
         )
         .filter(
           c =>
-            caseFilters.languages.size === 0 ||
-            c.languages.find(l => caseFilters.languages.has(l)),
-        )
-        .filter(
-          c =>
-            caseFilters.agency.size === 0 ||
-            (c.adjudicating_agency &&
-              caseFilters.agency.has(c.adjudicating_agency)),
+            agencyFilters.size === 0 ||
+            (c.adjudicating_agency && agencyFilters.has(c.adjudicating_agency)),
         )
         .filter(c =>
-          caseFilters.countries.size > 0
-            ? c.country && caseFilters.countries.has(c.country)
+          countriesFilters.size > 0
+            ? c.country && countriesFilters.has(c.country)
             : true,
```

Comment:
nit: is there a reason why this filter isn't of the format: 
`countriesFilters.size === 0 || (c.country && countriesFilters.has(c.country))`
like the rest of the filters?

Reply:
good point, ill fix it